src/main/webapp/js/stats-page-loader.js

```diff
@@ -1,27 +1,27 @@
 // Fetch stats and display them in the page.
-  function fetchStats() {
-    const url = '/stats';
-    fetch(url).then((response) => {
-      return response.json();
-    }).then((stats) => {
-      const statsContainer = document.getElementById('stats-container');
-      statsContainer.innerHTML = '';
+function fetchStats() {
+  const url = '/stats';
+  fetch(url).then((response) => {
```

Comment:
Nit:

```suggestion
  fetch(url).then((response) => response.json())
```

It's just a shorter version. We can define a function like:
```
const f = (s) => "s: " + s;
```
which is equivalent to
```
const f = (s) => {
  return "s: " + s;
};